frontend/src/components/uam/helpers/get-columns-helper/get-columns.helper.ts

```diff
@@ -1,25 +1,25 @@
 import { UsersGetAllItemResponseDto } from 'common/types/types';
 import { Column } from 'react-table';
 
-import { UserTableHeader } from '../../common/enums/enums';
+import { UserTableAccessor, UserTableHeader } from '../../common/enums/enums';
```

Comment:
```suggestion
import { UserTableAccessor, UserTableHeader } from 'common/enums/enums';
```

Reply:
Seems like this enums are correctly imported from `uam` local `common` (`src/components/uam/common/enums`) folder,, not from `src` `common` folder (`src/common/enums`).
Or you mean they should be moved to `src/common/enums`?

Reply:
You are right about first case

Reply:
@WilhelmWesser can you change it please)

Reply:
Change it back? :)

Reply:
Almost, just add absolute path)